api-examples/classify-text.py

```diff
@@ -1,12 +1,15 @@
 import baseline as bl
 import argparse
 import os
+from baseline.utils import str2bool
+
 parser = argparse.ArgumentParser(description='Classify text with a model')
-parser.add_argument('--model', help='A classifier model', required=True, type=str)
+parser.add_argument('--model', help='A directory containing classifier model files', required=True, type=str)
 parser.add_argument('--text', help='raw value', type=str)
 parser.add_argument('--backend', help='backend', default='tf')
 parser.add_argument('--remote', help='(optional) remote endpoint', type=str) # localhost:8500
 parser.add_argument('--name', help='(optional) service name', type=str)
+parser.add_argument('--preproc', help='(optional) set to true if want to use preproc', type=str2bool, default=False)
```

Comment:
Shouldnt this be an `exporter_type` not a `preproc` option?  Right now it seems really tightly coupled to a very specific addon, the other way would be more general

Reply:
`export.py` has an `exporter-type` option. this option is not tied to an addon. it just tells the `predict` methods to use/ not use preproc.